emscripten.py

```diff
@@ -273,14 +273,15 @@ def save_settings():
         const = const[1:-1]
       const = '{ ' + const + ' }'
       args = []
-      arity = metadata['asmConstArities'][k]
+      arity = metadata['asmConstArities'][k][-1]
```

Comment:
please add a comment saying that since this is sorted, that is the largest value.

actually - how about having it not sorted, and just doing `max()` here? seems simpler i think.